ui/DesignSystem/Primitive/Icon/Anchor.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import IconWrapper from "./IconWrapper.svelte";
+
+  export let dataCy = "";
+  export let style = "";
```

Comment:
Another way to write this is
```suggestion
  export let dataCy: string | undefined = undefined;
  export let style: string | undefined = undefined;
```

This has the following advantage: If this Icon is used without specifying the attributes they will be undefined and they will not be set on `IconWrapper`. Down the line, this means we don’t end up with empty attributes like `<div dataCy>` in the DOM. (In this specific case, we’ll need to fix `IconWrapper` though first).

As a general rule I’d prefer to use `undefined` as the default value.

Reply:
I totally agree, in this case I just copied an old component and forgot to update the prop types. Might be worth doing this for all icons in a separate follow-up.

e3c3eacfca73580c324d5d60f54c04e04069f9db